benchmark/single-source/SetTests.swift

```diff
@@ -334,6 +610,98 @@ public let SetTests = [
     tags: [.validation, .api, .Set],
     setUpFunction: { blackHole([setP, setQ]) }),
 
+  BenchmarkInfo(
+    name: "Set.subtracting.Sequence.Empty.Int",
+    runFunction: { n in run_SetSubtractingSequenceInt(setE, arrayAB, 0, 1000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setE, arrayAB]) }),
+  BenchmarkInfo(
+    name: "Set.subtracting.Sequence.Int.Empty",
+    runFunction: { n in run_SetSubtractingSequenceInt(setAB, arrayE, countAB, 1000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayE]) }),
+  BenchmarkInfo(
+    name: "Set.subtracting.Sequence.Empty.Box",
+    runFunction: { n in run_SetSubtractingSequenceBox(setOE, arrayOAB, 0, 1000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOE, arrayOAB]) }),
+  BenchmarkInfo(
+    name: "Set.subtracting.Sequence.Box.Empty",
+    runFunction: { n in run_SetSubtractingSequenceBox(setOAB, arrayOE, countAB, 1000 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, arrayOE]) }),
+  BenchmarkInfo(
+    name: "Set.Subtracting.Sequence.Int0",
+    runFunction: { n in run_SetSubtractingSequenceInt(setAB, arrayCD, countAB, 10 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayCD]) }),
+  BenchmarkInfo(
+    name: "Set.Subtracting.Sequence.Box0",
+    runFunction: { n in run_SetSubtractingSequenceBox(setOAB, arrayOCD, countAB, 10 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, arrayOCD]) }),
+  BenchmarkInfo(
+    name: "Set.Subtracting.Sequence.Int25",
+    runFunction: { n in run_SetSubtractingSequenceInt(setAB, arrayBC, countA, 10 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setAB, arrayBC]) }),
+  BenchmarkInfo(
+    name: "Set.Subtracting.Sequence.Box25",
+    runFunction: { n in run_SetSubtractingSequenceBox(setOAB, arrayOBC, countA, 10 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setOAB, arrayOBC]) }),
+  BenchmarkInfo(
+    name: "Set.Subtracting.Sequence.Int50",
+    runFunction: { n in run_SetSubtractingSequenceInt(setXY, arrayYZ, half, 10 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setXY, arrayYZ]) }),
+  BenchmarkInfo(
+    name: "Set.Subtracting.Sequence.Int100",
+    runFunction: { n in run_SetSubtractingSequenceInt(setP, arrayQ, 0, 10 * n) },
+    tags: [.validation, .api, .Set],
+    setUpFunction: { blackHole([setP, arrayQ]) }),
+
+  BenchmarkInfo(
+    name: "Set.Filter.Int0",
+    runFunction: { n in run_SetFilterInt(setAB, size, 0, 10 * n)},
```

Comment:
Should I change `10 * n` instead of `1000 * n`?

Reply:
I don't understand what are these `Filter` benchmark variants doing here at all… @lorentey requested adding only benchmarks for generic variants of set methods that take `Sequence`, as argument, as far as I've seen.

Reply:
@lorentey Did you request filter benchmark for `Set`, too? If so, @Gumichocopengin8, you might want to look at [DictionaryCopy.swift](https://github.com/apple/swift/blob/master/benchmark/single-source/DictionaryCopy.swift#L35-L42), which stresses `Dictionary` construction with a particular emphasis on avoiding possible a quadratic performance degradation.

Reply:
> Some Set operations have generic variants that take a sequence -- e.g., `Set.isSuperset<S: Sequence>(of: S)`. These are currently not covered by benchmarks at all, even though we're planning to [improve them](https://github.com/apple/swift/pull/21300) soon! I'd very much welcome benchmarks that measure those.

My understanding is that he also request filter benchmark because filter is also in the list.